repose-aggregator/src/docs/asciidoc/release-notes.adoc

```diff
@@ -13,6 +13,16 @@
 *** https://github.com/rackerlabs/api-checker/blob/api-checker-2.4.0/RELEASE.md[API Checker v2.4.0 release notes]
 ** Attribute Mapper: 1.3.0 → 1.3.1
 *** https://github.com/rackerlabs/attributeMapping/blob/attribute-mapper-1.3.1/RELEASE.md[Attribute Mapper v1.3.1 release notes]
+* https://repose.atlassian.net/browse/REP-5727[REP-5727] - Extracted trace ID logging to its own named logger.
+** The `trace-id-logging` Logger in your Log4j2 configuration will determine the logging behavior for trace ID logging.
+If the `trace-id-logging` Logger has not been configured, it will inherit the `Root` logger's configuration.
++
+[IMPORTANT]
+====
+If Repose is already installed and configured, and the `Root` logger is not configured to log at the `trace` logging level, then upgrading Repose will drop some or all trace ID logging messages.
+Please add and configure the `trace-id-logging` Logger to regain those messages.
+====
+
```

Comment:
So the naive approach could change logging behavior underneath people that upgrade, but already have a `log4j2` configuration in place. It is somewhat likely that they will lose at least the trace level trace ID log messages. Does that have some implication with regards to the version of the release this work can be in?